examples/unitTestExample/build.gradle

```diff
@@ -32,6 +32,8 @@ android {
 
 
 dependencies {
+    testCompile 'io.reactivex:rxjava:1.1.0'
```

Comment:
Without this, we will encounter `javassist.NotFoundException`. I don't know why this happens, but it is needed.